pro_tes/ga4gh/tes/task_runs.py

```diff
@@ -491,7 +508,16 @@ def _update_doc_in_db(
         tes_uri: str,
         remote_task_id: str,
     ) -> DbDocument:
-        """Update the document in the database."""
+        """Update the document in the database.
```

Comment:
Better (more concise): "Update the database document". Also, briefly looking at the code, this does not look like a _generic_ database document update method. Rather, you are updating something specific here, namely information about the TES endpoint that the task is forwarded to (going to be forwarded to or _has been_ forwarded to?), and a couple of other things.

Please try to be more descriptive/precise. What kind of update does this method really do?

Reply:
This function sets the end time( in TesTask.logs), sets the TES endpoint where the task is forwarded to and its remote task id ( in TesTask.logs.metadata), and then finally inserts the updated document in DB.

Reply:
will update its docstring accordingly in next commit.

Reply:
Done